jekyll/_cci2/env-vars.md

```diff
@@ -16,6 +16,11 @@ This document describes using environment variables in CircleCI in the following
 
 ## Overview
 
+CircleCI uses Bash,
```

Comment:
Move after the first paragraph as this is more specific about how to fill in the field on the page.